### PR TITLE
Add Integration Test for Method Invoke

### PR DESCRIPTION
**Pull request by JuanJose-Herrera**
# Description

Add Integration Test for Method Invoke

## Issue reference

https://github.com/dapr/java-sdk/issues/100

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: sdk-tests/pom.xml [hunk 1]
```diff
@@ -65,6 +65,12 @@
       <version>2.2.2.RELEASE</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.javafaker</groupId>
```
Comment (artursouza) [inline sdk-tests/pom.xml]: This seems to be a convenient package to generate data. We've finished a review of 3rd party dependencies. I do not see this one adding enough value to justify adding another 3rd party dependency.

File: sdk-tests/src/test/java/io/dapr/it/methodinvoke/http/MethodInvokeController.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    public void deleteMessages(@PathVariable Integer messageId){
+        messagesReceived.remove(messageId);
+    }
+
+    @GetMapping(path = "/messages")
+    public Map<Integer, String> getMessages() {
+        return messagesReceived;
+    }
+
+
```
Comment (artursouza) [inline sdk-tests/src/test/java/io/dapr/it/methodinvoke/http/MethodInvokeController.java]: Excessive spacing.

File: sdk-tests/pom.xml (hunk 1 above)
Comment (JuanJose-Herrera) [reply sdk-tests/pom.xml]: That was removed

File: sdk-tests/src/test/java/io/dapr/it/methodinvoke/http/MethodInvokeController.java (hunk 2 above)
Comment (JuanJose-Herrera) [reply sdk-tests/src/test/java/io/dapr/it/methodinvoke/http/MethodInvokeController.java]: spaces removed

**Pull request merged**